sparse_ho/tests/test_elastic.py

```diff
@@ -13,61 +13,57 @@
 from scipy.sparse import csc_matrix
 
 
-n_samples = 10
-n_features = 20
+n_samples = 100
+n_features = 100
 n_active = 5
-tol = 1e-16
-max_iter = 50000
 SNR = 3
 rho = 0.1
-X_train, y_train, beta_star, noise, sigma_star = get_synt_data(
-    dictionary_type="Gaussian", n_samples=n_samples,
+
+X, y, beta_star, noise, sigma_star = get_synt_data(
+    dictionary_type="Toeplitz", n_samples=n_samples,
     n_features=n_features, n_times=1, n_active=n_active, rho=rho,
     SNR=SNR, seed=0)
-X_train = csc_matrix(X_train)
-X_test, y_test, beta_star, noise, sigma = get_synt_data(
-    dictionary_type="Gaussian", n_samples=n_samples,
-    n_features=n_features, n_times=1, n_active=n_active, rho=rho,
-    SNR=SNR, seed=1)
-X_test = csc_matrix(X_test)
-X_val, y_val, beta_star, noise, sigma = get_synt_data(
-    dictionary_type="Gaussian", n_samples=n_samples,
-    n_features=n_features, n_times=1, n_active=n_active, rho=rho,
-    SNR=SNR, seed=2)
-X_val = csc_matrix(X_val)
-alpha_max = (X_train.T @ y_train).max() / n_samples
+X_s = csc_matrix(X)
+
+idx_train = np.arange(0, 50)
+idx_val = np.arange(50, 100)
+
+alpha_max = (X[idx_train, :].T @ y[idx_train]).max() / n_samples
```

Comment:
/ len(idx_train)   and use norm(ord='inf') (an abs is lacking here)

Reply:
np.abs missing